src/test/java/com/google/firebase/auth/multitenancy/TenantAwareFirebaseAuthIT.java

```diff
@@ -247,7 +258,7 @@ public void testVerifyTokenWithWrongTenantAwareClient() throws Exception {
       fail("No error thrown for verifying a token with the wrong tenant-aware client");
     } catch (ExecutionException e) {
       assertTrue(e.getCause() instanceof FirebaseAuthException);
-      assertEquals(FirebaseUserManager.TENANT_ID_MISMATCH_ERROR,
+      Assert.assertEquals("tenant-id-mismatch",
```

Comment:
Use a static import for `Assert.assertEquals`?

Reply:
Done